applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -114,6 +114,11 @@ Configuration options
 
 Check and configure the following configuration options for the sample:
 
+.. option:: CONFIG_SLM_CUSTOMIZED
+
+    This flag can be used to enable customized functionality
+    To add your own custom logic, enclose the code by ``#if defined(CONFIG_SLM_CUSTOMIZED)`` and enable this flag.
+
 .. option:: CONFIG_SLM_NATIVE_TLS
```

Comment:
```suggestion
.. option:: CONFIG_SLM_NATIVE_TLS - Use Zephyr mbedTLS
```
(I know this isn't part of the PR, but it's still wrong in the docs 😉)

Reply:
Sorry I did not noticed that before, now fixed